### PR TITLE
Update sensitive-information-type-entity-definitions.md

### PR DESCRIPTION
**Pull request by pratjain523**
@chrfox  Since most customers directly land on this page for SITs, added a link to the "https://docs.microsoft.com/en-us/microsoft-365/compliance/sensitive-information-type-learn-about?view=o365-worldwide" page here so that customers can navigate to this new page we added and learn about confidence levels and structure of an SIT  

Also added the definition of 3 new SITs

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 1]
```diff
@@ -151,6 +139,89 @@ A DLP policy has medium confidence that it's detected this type of sensitive inf
...
+## Argentina Unique Tax Identification Key (CUIT/CUIL)
+
+### Format
+
+Eleven digits with dash
+
+### Pattern
+
+Eleven digits with a dash:
+- two digits in 20, 23, 24, 27, 30, 33 or 34
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md:151]:
```suggestion
- two digits in 20, 23, 24, 27, 30, 33, or 34
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 2]
```diff
@@ -151,6 +139,89 @@ A DLP policy has medium confidence that it's detected this type of sensitive inf
...
+
+- Clave Unica de Identificacion Tributaria
+- CUIT
+- unique code of labour identification 
+- Clave Única de Identificación Tributaria
+- unique labour identification code
+- CUIL
+- Unique Tax Identification Key
+- Unique Labour Identification Key
+- Unique Key of Labour Identification
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md:195]:
```suggestion
- Unique Key of Labor Identification
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 3]
```diff
@@ -11689,6 +11760,132 @@ A DLP policy has low confidence that it's detected this type of sensitive inform
...
+
+- one digit between 1 to 9
+- one letter excluding  S, L, O, I, B, Z
+- one digit or letter excluding S, L, O, I, B, Z
+- one digit
+- an optional Hyphen
+- one letter excluding  S, L, O, I, B, Z
+- one digit or letter excluding S, L, O, I, B, Z
+- one digit
+- an optional Hyphen
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md:11780]:
```suggestion
- an optional hyphen
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 4]
```diff
@@ -11689,6 +11760,132 @@ A DLP policy has low confidence that it's detected this type of sensitive inform
...
+
+eleven character alphanumeric pattern
+  
+### Pattern
+
+- one digit between 1 to 9
+- one letter excluding  S, L, O, I, B, Z
+- one digit or letter excluding S, L, O, I, B, Z
+- one digit
+- an optional Hyphen
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md:11776]:
```suggestion
- an optional hyphen
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestions.

**Pull request merged**